examples/rn-connection-and-error/frontend/app/hooks/useDemoSyncTriggers.ts

```diff
@@ -147,6 +148,18 @@ export function useDemoSyncTriggers() {
     await currentUser.refreshCustomData();
   }, [currentUser]);
 
+  /**
+   * Trigger the user event listener by removing the user from the app.
+   */
+  const deleteUser = useCallback(() => {
+    // TODO: Update to use only `deleteUser`.
```

Comment:
Added this `TODO` until [#5327](https://github.com/realm/realm-js/issues/5327) is fixed.